spec/controllers/auth/registrations_controller_spec.rb

```diff
@@ -3,6 +3,45 @@
 RSpec.describe Auth::RegistrationsController, type: :controller do
   render_views
 
+  shared_examples 'checks for enabled registrations' do |path|
+    it 'redirects if it is in single user mode while it is open for registration' do
+      Fabricate(:account)
+      Setting.open_registrations = true
```

Comment:
I agree with that. However there are already similar codes and I followed them. I'll also fix them later.